topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorIT.java

```diff
@@ -654,4 +658,44 @@ public void testReconcile(TestContext context) {
     // TODO: What happens if we create and then change labels to the resource predicate isn't matched any more
     //       What then happens if we change labels back?
 
+
+    @Test
+    public void testKafkaTopicCreatedWithOwnerRef(TestContext context) {
+        String topicName = "test-kafka-topic-with-owner-ref";
+
+        // create a CM which will be owner reference for the topic
+        String cmName = "hodor";
+        HashMap<String, String> cmData = new HashMap<>();
+        cmData.put("strimzi", "rulez");
+        kubeClient.configMaps().inNamespace(NAMESPACE).create(new ConfigMapBuilder().withNewMetadata().withName(cmName)
+                .withNamespace(NAMESPACE).endMetadata().withApiVersion("v1").withData(cmData).build());
+        String uid = kubeClient.configMaps().inNamespace(NAMESPACE).withName(cmName).get().getMetadata().getUid();
```

Comment:
Please add a comment: `// this CM is created to be the owner of the KafkaTopic we're about to create.`